Source/WebKit/WebProcess/WebPage/gtk/WebPrintOperationGtk.cpp

```diff
@@ -278,6 +291,7 @@ void WebPrintOperationGtk::startPage(cairo_t* cr)
     cairo_surface_t* surface = cairo_get_target(cr);
     cairo_surface_type_t surfaceType = cairo_surface_get_type(surface);
     if (surfaceType == CAIRO_SURFACE_TYPE_PS) {
```

Comment:
The `CAIRO_SURFACE_TYPE` values are defined unconditionally, right?

Reply:
See https://gitlab.freedesktop.org/cairo/cairo/-/blob/master/src/cairo.h?ref_type=heads#L2565, this is defined in cairo.h.